runner/package.json

```diff
@@ -52,6 +52,7 @@
     "acorn": "7.1.1",
     "atob": "^2.1.2",
     "aws-sdk": "2.655.0",
+    "bell": "^10.0.0",
```

Comment:
I believe @hapi/bell will give you the most up to date version (12 something something)